tests/main/validate-container-happy/task.yaml

```diff
@@ -0,0 +1,37 @@
+summary: check the symlinks following the right track
+
+environment:
+    SNAP: test-snapd-validate-container-happy
+    SNAP_MOUNT_DIR: $(os.paths snap-mount-dir)
```

Comment:
Hm isn't this evaluated on the host? So if I use Fedora locally, but try to run the spread test on ubuntu-20.04-64, it will fail because a different mount dir will be expected?